samwise.js

```diff
@@ -1,23 +1,20 @@
-const capitalizedString = string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!